programs-service/src/models/mongoose/program.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const programSchema = new mongoose.Schema({
+    firstName: String,
```

Comment:
why do we need all these props like firstName, birthday etc here? I think it's a ctrl-c ctrl-v problem :)

Reply:
I will use this schema later when I know the list of properties. But yes, that was a copy-paste :)